app/javascript/components/copy-dashboard-form/copy-dashboard-form.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from 'patternfly-react';
+import MiqFormRenderer from '../../forms/data-driven-form';
+import createSchema from './copy-dashboard-form.schema';
+import { http, API } from '../../http_api';
+import handleError from '../../helpers/handle-failure';
+
+const CopyDashboardForm = ({ dashboardId }) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [initialValues, setInitialValues] = useState({});
+  const [schema, setSchema] = useState({});
+
+  const submitValues = (values) => {
+    miqSparkleOn();
+    const data = {
+      ...values,
+      dashboard_id: dashboardId,
+    };
+
+    return http.post(`/report/db_copy/${dashboardId}?button=save`, data, { skipErrors: [400] })
+      .then(() => miqAjaxButton('/report/dashboard_render'))
+      .catch(handleError, miqSparkleOff);
```

Comment:
i don't think that you can have multiple executors in promise callbacks(unless we have some polyfill). It will only execute the first one `handleError`.